### PR TITLE
Remove invalid slug

### PR DESCRIPTION
**Pull request by v-xuto**
According to [doc](https://review.learn.microsoft.com/en-us/help/platform/metadata-taxonomies?branch=main#product), remove invalid product slug `dapr`.

@hemarina, @greenie-msft for notification.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,6 @@ products:
 - azure
 - azure-container-apps
 - azure-service-bus
-- dapr
```
Comment (hemarina) [inline README.md:11]: If @greenie-msft want to keep this slug, let's submit a new slug request.

Comment (v-xuto) [reply README.md:11]: @greenie-msft  Any ideas?

**Review by hemarina — APPROVED** [review]

**Comment by hemarina** [comment]
@tonybaloney Would you mind review this PR?

File: README.md (hunk 1 above)
Comment (hemarina) [reply README.md:11]: Let's remove this slug for now and add it if needed in future.

**Review by tonybaloney — APPROVED** [review]

**Pull request merged**